doc/changes/changes_4.5.0.md

```diff
@@ -0,0 +1,8 @@
+# 4.5.0 - 2025-12-05
```

Comment:
Will you separately do a 8.29.13? ITDE is only up to 8.29.12 but docker-db dropped it.